docs/source/filesystems.rst

```diff
@@ -0,0 +1,33 @@
+Backend File-systems
+====================
+
+Fastparquet can use alternatives to the local disc for reading and writing parquet.
+
+One example of such a backend file-system is `s3fs <http://s3fs.readthedocs.io>`_, to connect to
+AWS's S3 storage. In the following, the login credentials are automatically inferred from the system
+(could be environment variables, or one of several possible configuration files.).
+
+.. code-block:: python
+
+    import s3fs
+    from fastparquet import ParquetFile
+    s3 = s3fs.S3FileSystem()
+    myopen = s3.open
+    pf = ParquetFile('/mybucket/data.parquet', open_with=myopen)
+    df = pf.to_pandas()
+
+The function `myopen` provided to the constructor must be callable with `f(path, mode)` and produce an open file context.
```

Comment:
In rst you'll need to use double backticks for code highlighting

```
`some text`  :  turns into italics
``some text`` : code syntax
```

This is a notable difference from markdown